Content.Server/Xenoarchaeology/XenoArtifacts/ArtifactSystem.cs

```diff
@@ -43,24 +40,28 @@ private void OnInit(EntityUid uid, ArtifactComponent component, MapInitEvent arg
     /// </summary>
     /// <remarks>
     /// General balancing (for fully unlocked artifacts):
-    /// Simple (1-2 Nodes): 1-2K
-    /// Medium (5-8 Nodes): 6-7K
-    /// Complex (7-12 Nodes): 10-11K
+    /// Simple (1-2 Nodes): ~10K
+    /// Medium (5-8 Nodes): ~30-40K
+    /// Complex (7-12 Nodes): ~60-80K
+    ///
+    /// Simple artifacts should be enough to unlock a few techs.
+    /// Medium should get you partway through a tree.
+    /// Complex should get you through a full tree and then some.
```

Comment:
this is for price not research points isnt it?

Reply:
yep. my bad